paper-ieee-trans.tex

```diff
@@ -487,13 +486,13 @@ \subsubsection{Quality metrics}
 \textit{Development metrics} are obtained programatically from several sources, namely GitHub
 API and Jenkins service, and graphically displayed as GitHub pages using GrimoireLab
 framework \cite{grimoirelab}. Per-component weekly reports, including the SQA requirement
-completeness, are issued and individually discussed through the different communication channels
+fulfilment, are issued and individually discussed through the different communication channels
```

Comment:
completeness was the term used in our KPI definition

Reply:
probably stick with what was defined (though I would prefer fulfillment)
and review English typos in the last version
I can do that in the final version

Reply:
well... maybe I've mised it in the deliverable :). I still vote for "fulfilment"

Reply:
agree

Reply:
surely I missed it in the deliverable.I still vote for "fulfilment"

Reply:
fulfilment then